grapple/types/structures.py

```diff
@@ -58,6 +62,12 @@ def __init__(self, of_type, *args, **kwargs):
                 f"{of_type} is not a subclass of DjangoObjectType and it "
                 "cannot be used with QuerySetList."
             )
+
+        # Enable in_menu for Page models.
+        if enable_in_menu is True and "in_menu" not in kwargs:
+            kwargs["in_menu"] = graphene.Argument(
+                graphene.Boolean, description=_("Filter by in menu.")
```

Comment:
This doesn't sound quite right to me, but I can't come up with a better string. "Filter by 'show in menus'" ?

Reply:
I got a bit more verbose, let me know what you think.